tests/unit_tests/test_client.py

```diff
@@ -399,11 +399,36 @@ def test_uc_get_variant():
     variant = unleash_client.get_variant("testVariations", context={"userId": "2"})
     assert variant["name"] == "VarA"
     assert variant["enabled"]
+    assert variant["is_feature_enabled"]
 
     # If feature flag is not.
     variant = unleash_client.get_variant("testVariations", context={"userId": "3"})
     assert variant["name"] == "disabled"
     assert not variant["enabled"]
+    assert not variant["is_feature_enabled"]
+
+    unleash_client.destroy()
+
+
+@responses.activate
+def test_uc_get_variant_feature_enabled_no_variants():
```

Comment:
Can we get away with bootstrapping rather than a mocked response? These sleeps are making the test suite insanely slow and I'm trying to be defensive about making that problem worse. There's some examples in the `test_client_specs.py` file where we use bootstrap over a mocked response that removes the need for sleeps and HTTP. If we can't then we can't but I gotta ask!

Reply:
Addressed in https://github.com/Unleash/unleash-client-python/pull/285/commits/7c687f0407dca73d47af776d805f0b682fb0bfff

Might be worth doing the same for the rest of the tests, but out of scope for this PR.

Reply:
I'd love to but gosh it's a lot of work. Definitely something on my back burner for when insomnia strikes again